doc/developer-guide/developer-guide.rst

```diff
@@ -4,59 +4,70 @@
 Build Cycle Developer Guide
 #########################################
 
-[This section should provide an introduction of the system to the developer and an brief overview of the CSC architecture.
-This should include links to background information where appropriate.]
+Build cycle is designed to provide a simple way of building containers for deployment at the various locations; NTS, TTS, summit and others.
```

Comment:
Maybe expand the acronyms NTS and TTS so it is clear for the reader what they mean?